### ISSUE TITLE
[Bug]: Change threshhold default for nominal/ordinal OR allow summing up ordinal (Likert) data

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-issues#2426]
### JASP Version

0.18.1

### JASP Module

Unrelated

### What analysis are you seeing the problem on?

data import / data wrangling

### What OS are you seeing the problem on?

Other

### Bug Description

![image](https://github.com/jasp-stats/jasp-issues/assets/4428185/4a39893d-5d08-4019-b32c-9db88f585b03)

The threshold is currently set to 10. 
But if I import a typical social sciences data set with many Likert scaled items, it would be much more user friendly, if the default threshold would be set to 4 (below the value of 5, which is the default for Likert scaled items).

Why?
Usually you will need to sum up many items with each other. But you cannot compute a sum with Likert scaled data, because it is always imported as ordinal. You have then to change the scale of dozens of items/variables from ordinal to scale - which is very inconvenient.
Another solution would be just to allow summing up ordinal data.

### Expected Behaviour

Summing up Likert scaled items should be easy.

**Comment by tomtomme** [comment]
I just read here: https://github.com/jasp-stats/jasp-issues/issues/1258

that SAV import ignores the threshold. This should be visible to the user / mentioned to the user next to the setting. If this cannot be changed easily then the only solution would be to allow + - / * for ordinal variables.

**Comment by tomtomme** [comment]
Still present in 0.18.2 flatpak-beta from 30.11.23

**Comment by tomtomme** [comment]
also related to 
https://github.com/jasp-stats/jasp-issues/issues/1421

**Comment by boutinb** [comment]
Maybe what we could do is a to add a button in Preferences to reload the data when the threshold is changed.

**Comment by tomtomme** [comment]
nice idea!
